LogAnalysis/HealthAnalysisModule/GetClusterGroup.psm1

```diff
@@ -0,0 +1,29 @@
+<#
+    The script block be used to parsing PS custom objects.
+    We can make some customized analysis operations for the PS custom objects in this script block.
+    The PS custom objects is stored in the script variable '$analyzeObject'.
+#>
+[ScriptBlock] $Global:analysisScript = `
```

Comment:
Make these as function and export from psm1, don't use global var

Reply:
I saw your pattern in invoking these scriptblocks in clixml.ps1. It's OK to keep the general analysis function and wrap one script from each module to call at once and give summary. 

But export a function first, and wrap the script on top of the function. In the future, some scenario may need to correlate between physical disks and virtual disks, so we can call function to get both psObjects and analyze together.